src/main/resources/db/migration/common/V1_204__alter_device_register_table.sql

```diff
@@ -0,0 +1,7 @@
+DROP SEQUENCE IF EXISTS der_seq;
+
+ALTER TABLE device_register DROP CONSTRAINT device_register_pk;
+
+ALTER TABLE device_register RENAME TO node_register;
+
+CREATE SEQUENCE nod_seq START WITH 1000;
```

Comment:
Why do we want to start the sequence at 1000?

Reply:
This has been reviewed and has been set at a much higher value 10,000, to account for existing data.